pkg/terraform/util.go

```diff
@@ -87,3 +87,31 @@ func CreateTerraformBackendFile(assetDir, data string) error {
 
 	return nil
 }
+
+// PrepareTerraformRootDir creates a directory named path including all
+// required parents.
+// An error is returned if the directory already exists.
+func PrepareTerraformRootDir(path string) error {
+	if err := os.MkdirAll(path, 0755); err != nil {
+		return fmt.Errorf("failed to create terraform assets directory at '%s', got: %w", path, err)
+	}
+
+	return nil
+}
+
+// PrepareLokomotiveTerraformModuleAt creates a directory named path
+// including all required parents and puts the Lokomotive Kubernetes
+// terraform module sources into path.
+// An error is returned if the directory already exists.
+//
+// The terraform sources are loaded either from data embedded in the
+// lokoctl binary or from the filesystem, depending on whether the
+// LOKOCTL_USE_FS_ASSETS environment variable was specified.
+func PrepareLokomotiveTerraformModuleAt(path string) error {
```

Comment:
In my opinion, `pkg/terraform` should have no relation with `assets` package. Terraform operates on directories in the filesystem. In my opinion, unpacking assets should be also done by `Cluster` object, which we currently don't have.